cmd/applyconfig/applyconfig.go

```diff
@@ -139,8 +140,20 @@ func (c *configApplier) asGenericManifest() error {
 	return err
 }
 
-func (c configApplier) asTemplate() error {
-	ocProcessCmd := makeOcCommand(ocProcess, c.path, c.user)
+func (c configApplier) asTemplate(params []templateapi.Parameter) error {
+	var args []string
+	for _, param := range params {
+		envValue := os.Getenv(param.Name)
+		if len(param.Generate) > 0 {
+			continue
+		}
+		if len(envValue) > 0 {
+			args = append(args, []string{"-p", fmt.Sprintf("%s=%s", param.Name, envValue)}...)
+		} else {
+			args = append(args, []string{"-p", fmt.Sprintf("%s=%s", param.Name, param.Value)}...)
```

Comment:
Is this branch even needed? Isn't `param.Value` used by default?

Reply:
Done.